src/main/java/com/google/sps/servlets/MemesServlet.java

```diff
@@ -87,5 +92,40 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String imgTag = String.format("<img src=\"%s\" width = \"300\"/>", blob.getMediaLink());
       response.getWriter().println(imgTag);
     }
+  }
+
+/** This funtions  randomly  get a string  captions from an array of string. When the function is called
```

Comment:
I like the descriptive blocks of comments and line comments you make before doing whatever it is you need to do.